auction-server/src/auction/repository/add_bid_analytics.rs

```diff
@@ -97,12 +118,12 @@ impl Repository {
                     searcher_token_mint: mint_searcher.to_string(),
                     searcher_token_amount: amount_searcher,
                     // TODO Fill this in
-                    searcher_token_usd_price: None,
+                    searcher_token_usd_price,
```

Comment:
one thought: the schema we decided upon doesn't have a field for `price_timestamp`, but i think it would be good to add that. that way, we can tell if any prices are stale. if it's not too hard to add, i would add that